pages/glossary/clearly-labeled-location.md

```diff
@@ -5,16 +5,18 @@ unambiguous: true
 objective: false
 ---
 
-Secondary information and alternative controls or functionality are often not displayed together with primary information or functionality. For example, an option to change a web page to dark mode may be placed on an options page instead of being available on every page and page state of a website. Another example is a maps application, where, instead of using GPS, an option is available in a dropdown menu to set the current location of the device. Such content should be placed in a clearly labeled location.
+Secondary information and alternative controls of functionality are often not displayed together with primary information or functionality. For example, an option to change a web page to dark mode may be placed on an options page instead of being available on every page and page state of a website. Another example is a maps application, where, instead of using GPS, an option is available in a dropdown menu to set the current location of the device. Such content should be placed in a clearly labeled location.
 
-The location of a target is said to be _clearly labeled_ when the target can be found by activating [instruments][] whose purpose, indicated by their [label][], leads the user to find the target or to another [page][web page] or page state from which this action can be repeated until the target is found.
+The location of a target is said to be _clearly labeled_ when the target can be found by activating [instruments][] whose "label" either leads the user to find the target, or to another [page][web page] or page state from which this action can be repeated until the target is found.
 
-Whether or not the content is "clearly labeled" depends on the starting point of the search. If page A has a link with a clear label to some piece of content, then the location of the content is clearly labeled. Page B, which can be in the same website, may not have such a link or may have a link with an ambiguous link text, and so the location of the content starting from page B is not clearly labeled.
+Whether or not the content is "clearly labeled" depends on the starting point of the search. If page A has a link with a clear "label" to some piece of content, then the location of the content is clearly labeled. Page B, which can be in the same website, may not have such a link or may have a link with an ambiguous link text, and so the location of the content starting from page B is not clearly labeled.
 
-**Note:** A [web page][] changes state when the [document's body][body] changes without a change in the [document's URL][URL].
+For the purpose of the definition of "Clearly labeled location", a "label" is any content with a textual alternative that identifies to a user an element with a [semantic role][] that inherits from `widget`.
```

Comment:
Does "identify" implies "descriptive name" or just "pointing at something"? (for me, "click the blue button if you dare" identifies `<button>The blue button</button>`)

"identifies to a user" => sighted user? AT user? Both? Others?

"textual alternative" => accessible name/description, or something else?

"any content with a textual alternative that …" => "(any content with a textual alternative) that …" or "any content (with a textual alternative that …)"? (what nominal group does "that" represent?)

Is this a "label"?
```html
<span>Click The link to go to the settings page</span>
<p>Lorem ipsum … (big wall of text containing other links)</p>
<a href="settings.html">The link</a>
```

I am not super confortable with creating yet a third meaning of label (after `label` element and the WCAG definition). Overloading names doesn't really help for being unambiguous…

I am not super confortable either with inlining the definition inside another one. This probably requires a full-fledged definition.

Reply:
@Jym77 I agree that we could try to avoid another meaning for "label". Given that this is used only inside this definition, I swapped "label" for "description". I also rephrased the sentence that defines "description" to try to address your concerns.

Regarding your example, I would not consider that to be a label because I don't think it allows the user to identify the widget that it refers to. But the definition is not objective, which means other people might argue that it identifies the widget.